portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -59,6 +53,19 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     response.sendRedirect("/index.html");
   }
 
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query("UserComments").addSort("timestamp", SortDirection.DESCENDING);
+    List<Comments> comments = new ArrayList<>();
```

Comment:
You can move the above line into readComments method. 
You json write can then just be
response.getWriter().println(gson.toJson(readComments(query)))

Ok to leave it for now. You can clean up in a follow up PR.

Reply:
Done (will push in a future PR)!